distributions/distribution-resources/src/main/resources/conf/icons/classic/readme.txt

```diff
@@ -0,0 +1,8 @@
+
+openHAB2 conf/icons/classic folder
+
+Your additional sitemap icons go here.
+Icons can be provided as png (32x32) or preferably as svg files.
+ClassicUI and BasicUI can be configured to accept svg (default) or png icons.
+
+Check out the openHAB2 documentation for more details: http://docs.openhab.org
```

Comment:
Would be cool if you were willing to write a page in the documentation (e.g. http://docs.openhab.org/configuration/icons.html) that we can specifically link to and which explains all that in detail.